src/v0/destinations/adobe_analytics/transform.js

```diff
@@ -76,6 +76,7 @@ const responseBuilderSimple = async (message, destinationConfig, basicPayload) =
     }
     payload.linkURL =
       adobeIntegrationsObject?.linkURL || context?.page?.url || 'No linkURL provided';
```

Comment:
'No linkURL provided' is this required? as this will not be in URL format

Reply:
Yes, this is standard when no url is given.